src/sentry/relay/config.py

```diff
@@ -124,6 +148,7 @@ def get_project_config(project, full_config=True, project_keys=None):
                 ],
                 "piiConfig": get_pii_config(project),
                 "datascrubbingSettings": get_datascrubbing_settings(project),
+                "features": get_exposed_features(project),
```

Comment:
Can only add this if there are features and otherwise leave it out. Alternatively, update failing tests to add `features: []` to the snapshots.